packages/framework/fluid-framework/src/index.ts

```diff
@@ -11,7 +11,7 @@
  */
 
 // ===============================================================
-// #region Public (and Beta) exports
+// #region Public, Beta and Alpha (non-legacy) exports
```

Comment:
Maybe also remove "Public exports" from the paired but no longer matching #endregion?

Reply:
Good catch, and done.